lua/hfcc/init.lua

```diff
@@ -23,6 +23,11 @@ M.setup = function(opts)
 
   config.setup(opts)
 
+  local api_token = config.get().api_token
+  if api_token == "" then
+    vim.notify("[HFcc] api token is empty, suggestion might not work", vim.log.levels.INFO)
```

Comment:
**suggestion**: we could lower the log message to `DEBUG` as this might be empty on purpose.
WDYT?

Reply:
Well I assume there would be an error message if the call doesn't succeed because of auth, so it makes sense to move this to `DEBUG`.

Maybe we should also change the default value of `api_token` to `null`.